src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -22,9 +13,48 @@ interface ItemProps {
   item: Conversation_conversation["items"][0]["item"]
 }
 
-const Item: React.FC<ItemProps> = props => {
+export type ItemType = "Artwork" | "Show"
+
+export const Item: React.FC<ItemProps> = props => {
   const { item } = props
-  if (item.__typename === "Artwork") {
+  if (item.__typename === "%other") return null
+  const itemType = item.__typename as ItemType
+
+  const getImage = item => {
+    if (itemType === "Artwork") {
+      return item.image?.url
+    } else if (itemType === "Show") {
+      return item.coverImage?.url
+    } else {
+      return null
+    }
+  }
+
+  const itemDetails = item => {
+    if (item.__typename === "Artwork") {
+      return [
+        <Sans key={1} size="4" weight="medium" color="white100">
+          {item.artistNames}
+        </Sans>,
+        <Sans key={2} size="2" color="white100">
+          {item.title} / {item.date}
+        </Sans>,
+      ]
```

Comment:
Instead of returning an array, you could return a fragment here

```suggestion
      return (
        <>
          <Sans size="4" weight="medium" color="white100">
            {item.artistNames}
          </Sans>
          <Sans size="2" color="white100">
            {item.title} / {item.date}
          </Sans>
        </>
``` 

It works either way, but you don't need to specify a key when you return it in a fragment.